content/posts/meetings.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
+title: Meetings
+permalink: /meetings/
+header_image: meetings
+---
+
+# Community Meetings
+NUTS maakt het mogelijk dat softwareapplicaties in de zorg met elkaar op een vertrouwelijke, veilige en toegankelijke manier kunnen communiceren. Tijdens onze maandelijkse communitymeetings praten wij je bij over Nuts en aan Nuts gerelateerde onderwerpen.   
+
+* Kom naar één van onze meetings (zie de [kalender](/kalender) voor tijden en locaties)
```

Comment:
Dit is nu een lijst met 1 bulletpoint. Beter weg te halen en dit stukje integreren met bovenstaande alinea.